drivers.json

```diff
@@ -28,6 +28,26 @@
       "Hardware-enforced Stack Protection Compatibility Test"
     ]
   },
+  "igb": {
+    "name": "Intel(R) 82576 Gigabit Dual Port Network Connection",
+    "device": "igb",
+    "type": 0,
+    "install_method": "no-drv",
+    "support": true,
+    "pretestcommands": [
+      {
+        "desc": "Rename igb ethernet adapter to SupportDevice0",
+        "run": "Rename-NetAdapter -Name (Get-NetAdapter -InterfaceDescription 'Intel(R) 82576 Gigabit Dual Port Network Connection').Name -NewName 'SupportDevice0'"
+      }
+    ],
+    "ignore_list": [
+      "NDISTest 6.5 - [2 Machine] - GlitchFreeDevice",
```

Comment:
GlitchFreeDevice - is it failing? Most probably not device issue because the driver should handle the amount of indicated packets, but if the card has some advanced internet moderation feature - could indicted that it is not working correctly.